spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,9 +323,46 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response(app_id: nil, platform: nil, review_id: nil, response: "")
+      data = {
+        responseText: response,
```

Comment:
[Question] If responseText is default value (empty string).
will API pass the value?